website/django/data/searchtest.py

```diff
@@ -224,7 +186,7 @@ def test_request_with_release_number(self):
 
         response_variant = response_data["data"][0]
 
-        self.assertEqual(response_variant["Data_Release_id"], new_variant_1.Data_Release_id)
+        self.assertEqual(response_variant["Data_Release_id"], new_variant_2.Data_Release_id)
```

Comment:
this should be something like: `self.assertEqual(response_variant["Data_Release_id"], {whatever_variant_you_ask_for}.Data_Release_id)`